.env.local.example

```diff
@@ -4,6 +4,7 @@
 AUTH_JWT_SECRET=""
 COINGECKO_API_KEY=""
 WALLET_CONNECT_PROJECT_ID=""
+SENTRY_DSN=""
```

Comment:
This `SENTRY_DSN` isn't a secret. It will appear in the public JavaScript bundle anyway. So you could just include it in the code 🤷‍♂️ Not that there's any harm in it, just in case that's why you made it an env var.

Reply:
On that note, we would at least want `NEXT_PUBLIC_SENTRY_DSN`, otherwise the [sentry.client.config.ts](https://github.com/defuse-protocol/defuse-widget/pull/16/files#diff-668b09e99b6037100b7526171c5b18292b5b22be0cf08b2a9dd55fbd64005dba) currently won't work.

Reply:
I see, I'll move it directly to configuration file.